--- sys/unix/Makefile.doc.orig	2016-10-02 20:41:46.620046458 -0500
+++ sys/unix/Makefile.doc	2016-10-02 20:42:50.714521302 -0500
@@ -26,10 +26,11 @@
 
 # Use the "cat" GUIDECMD if nroff and/or tbl and/or col are not installed
 # Not appropriate for creating Guidebook.txt.
-# GUIDECMD = cat Guidebook.txt
+GUIDECMD = cat Guidebook.txt
 # The following works better with groff-1.18, eg on Linux
 # GUIDECMD = $(GUIDE_PREFORMAT) | nroff -c -Tascii | $(COLCMD)
-GUIDECMD = $(GUIDE_PREFORMAT) | nroff | $(COLCMD)
+# GUIDECMD = $(GUIDE_PREFORMAT) | nroff | $(COLCMD)
+GUIDECMD = tbl tmac.n Guidebook.mn | nroff | $(COLCMD)
 
 # Only generate output for the current configuration:
 NHGREP = $(MAKEDEFS) --grep --input - --output -
